rosetta-source/src/main/resources/org/isda/codelist/xml/Disclaimer.txt

```diff
@@ -0,0 +1 @@
+This content is provided on an "as is" "as available" basis for illustrative purposes only. ISDA expressly disclaims all warranties, including the warranties of merchantability, fitness for a particular purpose and non-infringement. ISDA, FpML, and their directors, staff or agents, shall not be liable to you for any loss, injury, claim, liability or damage of any kind whatsoever resulting from, arising out of or in any way related to any errors in or omissions in this content. Any links to external data and sources are being provided as a convenience and for informational purposes only; they do not constitute an endorsement or an approval by ISDA. ISDA bears no responsibility for the accuracy, legality or content of the external site or for that of subsequent links. Contact the external site for answers to questions regarding its content.
```

Comment:
Is this necessary to include in a folder that is mostly meant to be read by a computer - not a human?